app/controllers/v0/preneeds/burial_forms_controller.rb

```diff
@@ -30,7 +30,7 @@ def burial_form_params
           :application_status, :has_attachments, :has_currently_buried, :sending_code,
           applicant: ::Preneeds::Applicant.permitted_params,
           claimant: ::Preneeds::Claimant.permitted_params,
-          currently_buried_persons: [::Preneeds::CurrentlyBuriedPerson.permitted_params],
+          currently_buried_persons: ::Preneeds::CurrentlyBuriedPerson.permitted_params,
```

Comment:
This is probably a dumb question: this is an array, why are we removing the brackets?

Reply:
it's already an array in `::Preneeds::CurrentlyBuriedPerson.permitted_params` so this was making it a double array